e2e/test/scenarios/embedding/embedding-snippets.cy.spec.js

```diff
@@ -158,6 +173,15 @@ features.forEach(feature => {
         .and("contain", "Ruby")
         .and("contain", "Python")
         .and("contain", "Clojure");
+
+      if (feature === "all") {
+        highlightedTexts().should("have.length", 1);
+
+        cy.findByRole("tab", { name: "Parameters" }).click();
+        cy.findByRole("tab", { name: "Look and Feel" }).click();
+
+        highlightedTexts().should("have.length", 1);
```

Comment:
Is this a duplicate of the above assertions? Just curious why it's duplicated, I guess it's because it's a different set of snippets?